src/lib.rs

```diff
@@ -37,4 +37,8 @@ pub struct __va_list_tag {
     pub reg_save_area: *mut ::std::os::raw::c_void,
 }
 
+#[cfg(feature = "use-bindgen")]
+include!(concat!(env!("OUT_DIR"), "/generated.rs"));
+
```

Comment:
Uhm, shouldn't this use `BINDINGS_OUT_DIR` if present (or `ALSA_SYS_BINDINGS_OUTPUT` given another comment)?